curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656d25e0c5d5aa11ade33754.md

```diff
@@ -0,0 +1,49 @@
+---
+id: 656d25e0c5d5aa11ade33754
+title: Task 150
+challengeType: 19
+dashedName: task-150
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+
+# --description--
+
+In the dialogue, Jake introduces himself to Sarah with the phrase `I'm Jake, from Security.` The word `from` is a preposition that shows the origin or source. In this context, `from Security` tells which department Jake works.
```

Comment:
Please replace "with the phrase" in the description with "saying". Also, @nieldakarla, how about having this question in English (backticks)?